hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionPolicy.java

```diff
@@ -13,44 +13,60 @@
     private static final int DEFAULT_RATE = 400;
     private static final int DEFAULT_MESSAGE_TTL = 3600;
     private static final int DEFAULT_MESSAGE_BACKOFF = 100;
+    private static final int DEFAULT_REQUEST_TIMEOUT = 1000;
+    private static final int DEFAULT_INFLIGHT_SIZE = 100;
```

Comment:
Property is still used, just not in the same way. Property CONSUMER_INFLIGHT_SIZE i used to dynamically force lower inflight size on all subscriptions without changing source code. DEFAULT_INFLIGHT_SIZE is just default value for consistency with rest of fields in SubscriptionPolicy. In fact this is default value when we read old subscription from zookeeper.